aptos-move/framework/move-stdlib/sources/option.move

```diff
@@ -244,6 +244,66 @@ module std::option {
         aborts_if false;
         ensures result == t.vec;
     }
+    /// Apply the function to the optional element, consuming it.
```

Comment:
Can we also document that this would do nothing (no-op) if the element is not present? Same for other new inline functions below